### ISSUE TITLE
python  -m hugchat.cli does nothing

### ISSUE THREAD

**Issue by akhil3417** [Soulter/hugging-chat-api#155]
running hugchat.cli does nothing , their is no log so that i could know whats going on

**Comment by Soulter** [comment]
> looks like cli() was never called

You are right. I have released  `v0.3.10` that contains your codes. Thank you!
